unity/CAVE/Assets/Editor/CLI.cs

```diff
@@ -283,9 +296,16 @@ private static void SetLinkActions()
             // Callback function when Debug.Log is called within the CLI script
             private static void HandleLog(string logString, string stackTrace, LogType type)
             {
-                // TODO 84: Change string based on LogType (rich color)
-                // Prepending "LOG:" will print the line to the screen (checked in Python script)
-                Console.WriteLine($"LOG:{logString}");
+                // Prepending "LOG:" will print the line to the screen (Python script)
+                string color = type switch
+                {
+                    LogType.Log => "white",
+                    LogType.Warning => "yellow",
+                    LogType.Error => "red",
+                    LogType.Exception => "bold red",
+                    _ => "white"
+                };
+                Console.WriteLine($"LOG:[{color}]{logString}[/{color}]");
```

Comment:
Do the colors make the input to the python script more difficult to process?

Reply:
No, `rich` picks up the syntax automatically so I don't have to worry about it in Python!

Reply:
Nice!